client/package.json

```diff
@@ -36,5 +36,8 @@
             "last 1 firefox version",
             "last 1 safari version"
         ]
+    },
+    "devDependencies": {
+        "eslint-config-prettier": "8.3.0"
```

Comment:
Why do we have the following `eslint-config` inside the `client/package.json` and not the `server/package.json`.  Should we have it in both?  Should we have it in the root `package.json`?  Again the `prettier` and `eslint` stuff is new to me, please correct me if I'm wrong.

Reply:
I think I added that to the client/package.json by mistake.

Reply:
Ok, let's go ahead and remove it for now and then we can go ahead and merge this in.

Reply:
@gleekzorp, it's been removed. I found npm-check, a package that looks through your project to show unused packages. I wanted to make sure the eslint-config-prettier wasn't the only package installed without use. There were a few that it said were unused that I know are not, because of how it checks if packages are referenced, so you have to confirm that the recommendation is accurate, but I was able to get rid of a few things and update a few things. And the site seems to be working properly. I did have to clear my cache to fix some errors, though. I would recommend, before approving this, pull it down, clear cache and test for a few minutes to make sure I didn't miss anything.

Reply:
I had the hardest time trying to get `node-sass` to work on my machine.  I was able to randomly get it to work once but when I tried to recreate what I did I wasn't able to make it work again.  I'm thinking maybe we move away from `node-sass`?  it seems as if it `deprecated` now.  https://www.npmjs.com/package/node-sass.  Obviously, it seems this is specific to my computer though seeming as you have been able to get this to work.

I went ahead and added `sass` instead of `node-sass` and I was able to get it to work and everything looks good.  Lets stick with `node-sass` for this pull request and I opened a new ticket to start a discussion on this `node-sass` deprecation at #89.

Accepting and merging now.